lib/crypto.js

```diff
@@ -330,11 +330,12 @@ function getRandomBytesAlias(key) {
 }
 
 ObjectDefineProperties(module.exports, {
-  // crypto.fips is deprecated. DEP0093. Use crypto.getFips()/crypto.setFips()
   fips: {
     __proto__: null,
-    get: getFips,
-    set: setFips,
+    get: deprecate(getFips, 'The crypto.fips is deprecated. ' +
+      'Please use crypto.getFips()', 'DEP0093'),
+    set: deprecate(setFips, 'The crypto.fips is deprecated. ' +
+      'Please use crypto.setFips()', 'DEP0093'),
```

Comment:
```suggestion
      'Use crypto.setFips() instead.', 'DEP0093'),
```

Reply:
Why? The wording is taken from the deprecations.md file.